backends/multidimensional_graphite.js

```diff
@@ -1,10 +1,47 @@
-var util = require('util');
-var events = require('events');
+var util = require('util')
+  , events = require('events')
+  , logger = require('../lib/logger')
+  , getHostname = require('os').hostname
+  , getBasename = require('path').basename;
 
 var log = console; // the logger
 var debug;
 
 
+function  createBackendLogger(config, logger) {
```

Comment:
I strongly support this change. Modifying the global logger was causing a lot of tests to fail and it's something you shouldn't really do after the initialization carried in the main entry point of the application. Moving this change to our custom backend file also isolates our code changes from the main codebase and is going to help us avoiding merge conflicts when updating or when pushing patches upstream (if ever).